rpcclient/infrastructure.go

```diff
@@ -1065,6 +1068,9 @@ type ConnConfig struct {
 	// Pass is the passphrase to use to authenticate to the RPC server.
 	Pass string
 
+	// Params is the string represening the network that the server is running.
+	Params string
```

Comment:
Mention that if not set, defaults to mainnet?

Reply:
Added a note